napalm/base/test/getters.py

```diff
@@ -24,6 +24,9 @@
 from napalm.base.test import conftest
 
 
+inspect_getargspec = getattr(inspect, "getfullargspec", inspect.getargspec)
```

Comment:
Can you just add a comment on why we need this? So we remember it in the future.